packages/flutter/test/widgets/editable_text_shortcuts_test.dart

```diff
@@ -90,33 +90,37 @@ void main() {
     );
   }
 
+  // Should we remove this test, because if this PR is approved it won't be possible
+  // to have an invalid selection for the controller if it is attached to a focused
+  // TextField.
```

Comment:
I think it's fine to delete the whole test!  Making a bug impossible is even better than having a test to catch it.